CHANGES.rst

```diff
@@ -377,6 +377,8 @@ tweakreg
 
 - Bug fix: Use integer division in Python 3. [#3072]
 
+- Bug fix: Improved 2D Histogram (pre-match shift) algorithm in Python. [#3281]
```

Comment:
Can you move this up to the 0.13.2 section?  It should be at the top of the file instead of down here.

Reply:
@jdavies-st Thanks for noticing this! Done.